### ISSUE TITLE
Use on allies?

### ISSUE THREAD

**Issue by Altalus** [EasyFarm/EasyFarm#247]
This might completely be outside the scope of your tool/project, but as I am trying to get rid of multiple bots I was wondering, for healing and enhancing especially, if there was in you TODO list the possibility of the bot having another target, party members?

**Comment by Mykezero** [comment]
Party support is something I have been wanting for really long time now. 

There's actually an existing branch for party healing already with most of the healing functionality implemented which may only need a little bit of polishing. 

https://github.com/EasyFarm/EasyFarm/blob/3ea49258e5958b236641113b24994773f112b55a/EasyFarm/States/HealingState.cs#L54

Party buffing would take a little bit longer to add since that requires keeping track of who's already buffed. 

Thankfully, most of the bots for FFXI these days are open source, so I definitely have a lot of resources to draw ideas from ^^;

**Comment by Altalus** [comment]
Indeed, open source is awesome, just be sure to give credits to the original authors/project !

 

The healer bot I am using is “Cure Please!” found here: 

 

-          Original port (port done by Wiccaan): http://www.elitemmonetwork.com/forums/viewtopic.php?f=11&t=509

-          New BETA version (By Daniel_H): http://www.elitemmonetwork.com/forums/viewtopic.php?f=10 <http://www.elitemmonetwork.com/forums/viewtopic.php?f=10&t=627> &t=627

 

Source seems to be here: https://github.com/DanielHazzard/Cure-Please

 

Hope this helps !

______________________________

Guillaume Gingras
